backend/infrahub/events/branch_action.py

```diff
@@ -36,20 +40,23 @@ def get_messages(self) -> list[InfrahubMessage]:
         return events
 
 
-class BranchCreateEvent(InfrahubBranchEvent):
+class BranchCreatedEvent(InfrahubEvent):
     """Event generated when a branch has been created"""
 
-    branch_id: str = Field(..., description="The ID of the mutated node")
+    event_name: str = f"{EVENT_NAMESPACE}.branch.created"
```

Comment:
Do we want to have this as a regular attribute? Not a @property or computed field? Will there be a scenario where we want to override this for a specific event at any point?

Reply:
hard to tell for sure right now, let's revisit that once we have more event implemented